tests/signal/signal.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * tests/setup/sigterm.js
```

Comment:
Can you add a comment here (and at the top of the tests/signal/sigterm.js file) about why we're calling these tests from a separate script? Otherwise six months from now someone will try to simplify and just run these tests from test-main.

Reply:
In fact, they are running in the main test thread. The only one is the KUE.js - I have added the comment as suggested

Reply:
will add the signal folder to the main thread actually - sorry.